### PR TITLE
feat: tour de sol page implemented

### PR DESCRIPTION
**Pull request by manuel-calavera**

### CODE REVIEW

File: src/v2/components/TourDeSol/Table/index.jsx [hunk 1]
```diff
@@ -0,0 +1,114 @@
...
+          </Link>
+        </TableCell>
+        <TableCell>{row.stake}</TableCell>
+        <TableCell>TODO</TableCell>
+      </TableRow>
+    );
+  };
+  const renderCard = card => {
+    return (
+      <div
```
[the remark this answers is not in the document]
Comment (manuel-calavera) [reply src/v2/components/TourDeSol/Table/index.jsx]: @sunnygleason it is being used for both mobile and desktop breakpoints

File: src/v2/components/TourDeSol/Ranking/index.jsx [hunk 2]
```diff
@@ -0,0 +1,49 @@
...
+
+  const renderNode = node => {
+    const position = (node.stake * 100) / maxVal;
+    return (
+      <li key={node.nodePubkey} className={classes.item}>
+        <div className={classes.name}>{node.nodePubkey}</div>
+        <div className={classes.bar}>
+          <div
+            className={classes.icon}
+            style={{left: `calc(${position}% - 26px)`}}
```
[the remark this answers is not in the document]
Comment (manuel-calavera) [reply src/v2/components/TourDeSol/Ranking/index.jsx]: yes

File: src/v2/components/Validators/Detail/index.jsx [hunk 3]
```diff
@@ -135,8 +135,7 @@ const ValidatorsDetail = ({match}: {match: Match}) => {
   return (
     <Container>
       <div className={classes.root}>
-        <SectionHeader>
-          Validator Detail
+        <SectionHeader title=" Validator Detail">
```
[the remark this answers is not in the document]
Comment (manuel-calavera) [reply src/v2/components/Validators/Detail/index.jsx]: done

**Comment by sunnygleason** [comment]
@manuel-calavera thank you!
@mvines merging FYI ... let me know if we see any issues on edge

**Pull request merged**